Source/WebCore/page/Navigator+LoginStatus.idl

```diff
@@ -28,7 +28,6 @@
     EnabledBySetting=LoginStatusAPIEnabled,
     ImplementedBy=NavigatorLoginStatus
 ] partial interface Navigator {
-    Promise<undefined> setLoggedIn();
-    Promise<undefined> setLoggedOut();
+    Promise<undefined> setLoginStatus(DOMString status);
```

Comment:
Where is the spec for this?

I am surprised this is not a string enumeration in the spec:
- https://webidl.spec.whatwg.org/#idl-enums

Reply:
Yes, I think this should take a string enumeration instead. It also looks like the function name was changed to just `setStatus`.

https://fedidcg.github.io/FedCM/#login-status-javascript